docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+# Simple docker-composer.yml
+# You can change container_name, port or volume location
+
+version: '3.3'
+
+services:
+  uptime-kuma:
+    image: louislam/uptime-kuma:1
```

Comment:
You could also add a `build:`-property here, because you can then also use docker-compose for performing a docker build (`docker compose build`). It's documented [[here](https://docs.docker.com/compose/compose-file/compose-file-v3/#build)]